services/order.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import Ticket, Order, MovieSession
+
+
+def create_order(tickets: list,
+                 username: str,
+                 date: datetime = None) -> Ticket:

```

Comment:
The return type hint should be `list[Ticket]` instead of `Ticket` because `bulk_create` returns a list of created objects.